ipywidgets/static/widgets/js/manager.js

```diff
@@ -306,6 +306,22 @@ define([
 
     WidgetManager.prototype.create_model = function (options) {
         /**
+         * For backward compatibility. Custom widgets may be relying on the fact
+         * that create_model was creating a comm if none was provided in options.
+         * Unlike the old version of create_model, if no comm is passed,
```

Comment:
It is not backward incompatible because we could not pass model id in the options before.
The new create_model only *allows* setting the model id.